### PR TITLE
New user guide draft one

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

**Comment by mlocher** [comment]
Nice work, do you want to release that as is, or make changes additional changes?

(Staging build available at https://documentation.codeship.com/staging/new-user-guide/general/account/new-user-signup/)

**Comment by ethangj** [comment]
@mlocher will release if you approve.

File: _general/account/new-user-signup.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+If you're looking to sign up and create a project on Codeship, this guide will walk you through the initial onboarding process.
+
+### Signing Up
+
+**Sign up** [Navigate to our signup page](https://codeship.com/registrations/new).  You can sign up either via your Source Code Management (SCM) or email address.  SCMs we currently support are: GitLab, Bitbucket, or GitHub.
+
+![SCM Choice]({{ site.baseurl }}/images/new-user-setup/scm-choice.png)
+
+- **Via GitHub:**  After selecting GitHub, you are automatically navigated to a login page where you must enter your GitHub login credentials in order to give Codeship access to your account.
```
Comment (mlocher) [inline _general/account/new-user-signup.md]: I would emphasis that those login pages are provided by the SCM provider and we don't get to see (or store) the credentials for those services at all.

File: _general/account/new-user-signup.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+![GitLab Login]({{ site.baseurl }}/images/new-user-setup/gl-login.png)
+
+- **Via Email:**   If you prefer to not sign up for Codeship via an SCM, you may also sign up via email.  You simply provide your: Name, Email Address, and create a Password, then hit ‘Sign up for free’.
+
+![Email Login]({{ site.baseurl }}/images/new-user-setup/email-login.png)
+
+*You can learn more about the permissions we ask when authenticating with SCM [here](https://documentation.codeship.com/general/account/permissions/).  Learn more about security at Codeship [here](https://documentation.codeship.com/general/about/security/).*
+
+### New User setup
```
Comment (mlocher) [inline _general/account/new-user-signup.md]: Should _setup_ have a capital _S_ as well?

File: _general/account/new-user-signup.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+![Email Login]({{ site.baseurl }}/images/new-user-setup/authenticate-scm.png)
+
+Simply click on the SCM icon you would like to connect with and proceed to provide your login credentials in order to give Codeship access to your account (see step 1 for further information).
+
+**Choose your repository:** Once you have authenticated your SCM you have to select your repository.  To choose your repository, simply paste your Git clone URL into the provided area and click ‘Connect’.  
+
+_Examples_:
+- git@gitlab.com:<username>/<repository_name>.git
+-https://username@gitlab.com/<username>/<repository_name>.git
```
Comment (mlocher) [inline _general/account/new-user-signup.md]: missing space between the dash and https

File: _general/account/new-user-signup.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+![Gitlab Repo]({{ site.baseurl }}/images/new-user-setup/gl-repo.png)
+
+Now that you’ve connected your repo, it’s time to select your infrastructure!  You simply have to decide if you want to set up your project with [Codeship's Basic Infrastructure.](https://codeship.com/features/basic) or [Codeship's Pro Infrastructure.](https://codeship.com/features/pro)
+
+
+**Basic** is a good place to start if:
+
+- You want out of the box configuration
+- Pre-installed CI dependencies
```
Comment (mlocher) [inline _general/account/new-user-signup.md]: I would reword those bullet points because _Basic is a good place to start if pre-installed CI dependencies_ isn't a working sentence (whereas the first bullet point is).

File: _general/account/new-user-signup.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+- Fully customizable CI environment
+- Local build runner for test consistency
+
+Once you know which type of infrastructure you’d like to use, simply click ‘Select Infrastructure’ on the corresponding project.
+
+![Select Codeship Infrastructure]({{ site.baseurl }}/images/new-user-setup/select-infra.png)
+
+## For More Information
+
+If you want to learn more please contact us at [helpdesk@codeship.com](mailto:helpdesk@codeship.com).
```
Comment (mlocher) [inline _general/account/new-user-signup.md]: Should we point them to the web-based helpdesk, or do we think we will get questions from people who didn't create an account yet?

**Comment by ethangj** [comment]
@mlocher Addressed!
